添加老饭骨视频描述中固定部分的翻译 (#636)
immoonancient/YTSubtitles: subtitles/lao-fan-gu/README.md

https://www.youtube.com/channel/UCBJmYv3Vf_tKcQr5_qmayXg

## 人物

| 称呼 | 翻译 |
| ---- | ---- |
| 大爷 | Da Ye |
| 二伯 | Er Bai |
| 三叔 | San Shu |

> 

Author: ```suggestion
| 师父 | master |
| 徒弟 | disciple |
```

## 视频描述中的固定部分

中文
```
大家好，我們是老飯骨！三個能吃、會做、擅長侃大山的北京大爺。
我們的菜好看好吃又硬核，過去我們都是做國宴打美食外交的，如今退休在家和老朋友壹起想把我們年輕時研究操練的菜品分享給大家，讓大家在家裏學著做真正兒的大菜，嘗嘗國宴菜品和中外領導人吃過的美食。
我們分別是大爺鄭秀生 國宴大師北京飯店行政總廚，親自主理建國五十周年，開國第壹宴工作；
二伯孫立新；多次擔任多位國家領導人宴請中外貴賓的菜單主設計者，
三叔武劍利，國際高級烹飪技師，中華金廚，是著名的“熏肉之鄉”柴溝鋪的第三代傳人，研發了“武式熏醬”大招。
三位老飯骨（吃貨）合體教妳做地道美味的家常菜，最正宗的北方菜，還有幾乎失傳的老菜，家傳秘方通通傾囊相授！
我們會每周壹三五更新視頻，教妳做菜，和妳嘮嗑，與妳分享家的味道。
喜歡我們的朋友歡迎評論，點贊和訂閱，打開小鈴鐺不會錯過我們的最新動態哦！
點擊下面的鏈接可以直接訂閱哦！
https://bit.ly/30QmA1X
*Facebook*
https://www.facebook.com/laofangu/
```

英文
```

> Hello, everyone, we are Lao Fan Gu, three old men from Beijing that can eat, cook and excel at chatting.

Author: ```suggestion
Hello everyone! We are Lao Fan Gu, three old men from Beijing that excel at eating, cooking and chit-chatting.
```

> Our dishes are good-looking, delicious and hardcore. All three of us used to work for Chinese National Banquet catered to government guests. Now we retire at home with old friends and want to share the dishes we studied and practiced when we were young, so everyone can try at home what country leaders are catered when they visit China. 

Reviewer: ```suggestion
Our dishes are good-looking, delicious and hardcore. All three of us used to work as chefs of China State Banquet catered to government guests. Now we retire at home and together with our old friends, we want to share the dishes we studied and practiced when we were young, so everyone can prepare them at home and try what country leaders are catered when they visit China. 
```

Reviewer: 1 work for是给人物/公司 work at为公司 work in是公司部门/大的地区 没有合适的就写成as chef 了
2 国宴是state banquet  谷歌可获答案，也可分为state lunch or state dinner
3 retire at home with old friends 会以为是和朋友在家退休hhh 和原句有偏离
4 try at home what xxx are catered ，个人感觉不通顺

Author: ```suggestion
Our dishes are good-looking, delicious and hardcore. All the three of us used to work as chefs of China state banquets catered to government guests. Now we are retired at home. Together with our old friends, we want to share the dishes we studied and practiced when we were young, so everyone can prepare them at home and try what country leaders are catered when they visit China.  
```
State banquet 不是专有名词

> Da Ye (Xiusheng Zheng): Executive Chef for Beijing Hotel, hosted China's 50th Anniversary dinner.

Author: ```suggestion
Da Ye (Zheng Xiusheng): Executive Chef of Beijing Hotel, hosted China's 50th Anniversary dinner.
```

> Er Bai (Lixin Sun): Designed dinner menus multiple times for Chinese National Banquet hosting foreign country leaders.

Reviewer: ```suggestion
Er Bai (Lixin Sun): Planned dinner menus multiple times for China State Banquet hosting foreign country leaders.
```

Author: ```suggestion
Er Bai (Sun Lixin): Planned dinner menus multiple times for China state banquets hosting foreign country leaders.
```

> San Shu (Jianli Wu):  Golden Cook of China. 3rd Generation successor of Caigoupu smoked meat. Developed 'Wu type smoked sauce'. 

Reviewer: ```suggestion
San Shu (Jianli Wu):  Golden Cook of China. 3rd Generation successor of Caigoupu smoked meat. Developed 'Wu type smoky sauce'. 
```

> We will upload new videos on Monday, Wednesday and Friday. 

Reviewer: ```suggestion
We will upload new videos on every Monday, Wednesday and Friday. 
```

Reviewer 2: ```suggestion
We will upload new videos every Monday, Wednesday and Friday. 
```

> You are welcome to comment, like and subscribe to our channel. Turn on the notification to receive the most updated status of the channel!

Reviewer: ```suggestion
You are welcome to comment, like and subscribe to our channel. Turn on the notification bell to receive the newest updates of the channel!
Click the link down below to subscribe to our channel!
```

三叔武劍利，國際高級烹飪技師，中華金廚，是著名的“熏肉之鄉”柴溝鋪的第三代傳人，研發了“武式熏醬”大招。
三位老飯骨（吃貨）合體教妳做地道美味的家常菜，最正宗的北方菜，還有幾乎失傳的老菜，家傳秘方通通傾囊相授！
我們會每周壹三五更新視頻，教妳做菜，和妳嘮嗑，與妳分享家的味道。
喜歡我們的朋友歡迎評論，點贊和訂閱，打開小鈴鐺不會錯過我們的最新動態哦！
點擊下面的鏈接可以直接訂閱哦！
https://bit.ly/30QmA1X
*Facebook*
https://www.facebook.com/laofangu/
```

英文
```
Hello everyone! We are Lao Fan Gu, three old men from Beijing that excel at eating, cooking and chit-chatting.
Our dishes are good-looking, delicious and hardcore. All the three of us used to work as chefs of China state banquets catered to government guests. Now we are retired at home. Together with our old friends, we want to share the dishes we studied and practiced when we were young, so everyone can prepare them at home and try what country leaders are catered when they visit China.  
Da Ye (Zheng Xiusheng): Executive Chef of Beijing Hotel, hosted China's 50th Anniversary dinner.
Er Bai (Sun Lixin): Planned dinner menus multiple times for China state banquets hosting foreign country leaders.
San Shu (Wu Jianli):  Golden Cook of China. 3rd Generation successor of Caigoupu smoked meat. Developed 'Wu type smoky sauce'. 
We will upload new videos every Monday, Wednesday and Friday. 

> You are welcome to comment, like and subscribe to our channel. Turn on the notification bell to receive the newest updates of the channel!

Author: ```suggestion
You are welcome to comment, like and subscribe to our channel. Turn on the notification bell to receive the latest updates of the channel!
```
